sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/application-changes-report-integration/application-changes-report-integration.scheduler.ts

```diff
@@ -25,6 +25,22 @@ export class ApplicationChangesReportIntegrationScheduler extends BaseScheduler<
     super(schedulerQueue, queueService);
   }
 
+  /**
+   * To be removed once the method {@link process} is implemented.
+   * This method "hides" the {@link Process} decorator from the base class.
+   */
+  async processQueue(): Promise<string | string[]> {
```

Comment:
Can we have the default implementation in the base class itself so that we don't  have to repeat in other places? - Is that approach not working?

Reply:
I found that when class(indluding subclass) decorated twice the bull doesn't accept it. 
![image](https://github.com/user-attachments/assets/3cd2c786-8df8-4bf9-a072-e773caadd767)